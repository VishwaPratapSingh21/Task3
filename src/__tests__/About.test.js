import React from 'react';
import { shallow, mount } from 'enzyme';
import toJson, { shallowToJson } from 'enzyme-to-json';


import About from '../components/auth/About';

describe('About component', () => {
  const wrapper = shallow(<About />);

  it('renders properly', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });

  it('it has a div', () => {
    expect(wrapper.find('div').length).toBe(1);
  });

  it('it has h1 and h4 headers', () => {
    expect(wrapper.find('h1').length).toBe(3);
    expect(wrapper.find('h4').length).toBe(3);
  });
});

