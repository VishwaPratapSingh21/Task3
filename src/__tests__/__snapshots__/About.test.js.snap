// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component renders properly 1`] = `
<div
  id="about"
>
  <h1>
    About Yummy Recipes
  </h1>
  <h4>
    It is a web app, built in React and it consumes a Flask API, It is a platform for, people who care about their nutrition go ahead and note down their recipes so as to keep practicing
  </h4>
  <h1>
    Why Yummy Recipes App
  </h1>
  <h4>
    Your food choices each day affect your health — how you feel today, tomorrow, and in the future. Good nutrition is an important part of leading a healthy lifestyle. Combined with physical activity, your diet can help you to reach and maintain a healthy weight, reduce your risk of chronic diseases (like heart disease and cancer), and promote your overall health.
  </h4>
  <h1>
    The Impact of Nutrition on Your Health
  </h1>
  <h4>
    Unhealthy eating habits have contributed to the obesity epidemic in the United States: about one-third of U.S. adults (33.8%) are obese and approximately 17% (or 12.5 million) of children and adolescents aged 2—19 years are obese.1 Even for people at a healthy weight, a poor diet is associated with major health risks that can cause illness and even death. These include heart disease, hypertension (high blood pressure), type 2 diabetes, osteoporosis, and certain types of cancer. By making smart food choices, you can help protect yourself from these health problems.
  </h4>
</div>
`;
